build.sc

```diff
@@ -53,7 +53,22 @@ object `integration-core` extends Module {
   }
 }
 
+object `integration-docker` extends CliIntegrationDockerCore {
```

Comment:
Do we really need that one? I think by renaming `modules/integration-docker` to `modules/integration/docker`, `integration.docker` below should automatically pick the right sources.

Reply:
Yes, we don't need this definition of module. I moved `modules/integration-docker` to `modules/integration/docker` and it correct pick the right sources.